server/issue_parser.go

```diff
@@ -21,13 +20,8 @@ func parseJiraLinksToMarkdown(text string) string {
 	return jiraLinkWithTextRegex.ReplaceAllString(text, "[${1}](${2})")
 }
 
-func mdKeySummaryLink(issue *jira.Issue) string {
-	// Use Self URL only to extract the full hostname from it
-	pos := strings.LastIndex(issue.Self, "/rest/api")
-	if pos < 0 {
-		return ""
-	}
-	return fmt.Sprintf("[%s](%s%s)", issue.Key+": "+issue.Fields.Summary+" ("+issue.Fields.Status.Name+")", issue.Self[:pos], "/browse/"+issue.Key)
+func mdKeySummaryLink(issue *jira.Issue, instance Instance) string {
+	return fmt.Sprintf("[%s](%s%s)", issue.Key+": "+issue.Fields.Summary+" ("+issue.Fields.Status.Name+")", instance.GetJiraBaseURL(), "/browse/"+issue.Key)
```

Comment:
I know this was already here but I think we should make this change to make it a little easier to read:

```suggestion
	return fmt.Sprintf("[%s: %s (%s)](%s%s)", issue.Key, issue.Fields.Summary, issue.Fields.Status.Name, instance.GetJiraBaseURL(), "/browse/"+issue.Key)
```